packages/hcs/tests/hedera-hcs-service.test.ts

```diff
@@ -0,0 +1,286 @@
+import { PrivateKey } from '@hashgraph/sdk'
+import { HederaHcsService } from '../src/hedera-hcs-service'
+import { getRandomStr } from './utils/utils'
+
+const operatorId = process.env.HEDERA_TESTNET_OPERATOR_ID ?? ''
+const operatorKey = process.env.HEDERA_TESTNET_OPERATOR_KEY ?? ''
+
+describe('Hedera HCS Service', () => {
+  jest.setTimeout(1200000);
```

Comment:
Do we still want to set such a timeout here and set timeout directly from tests in general?
If not, please double-check similar places.

Reply:
Let's check if we can conveniently set it globally, otherwise keep it as is.